packages/cli/src/commands/test.js

```diff
@@ -86,22 +110,22 @@ export const handler = async ({
   }
 
   if (sides.includes('api')) {
-    args.push('--runInBand')
+    jestArgs.push('--runInBand')
   }
 
-  args.push(
+  jestArgs.push(
     '--config',
     `"${require.resolve('@redwoodjs/core/config/jest.config.js')}"`
   )
 
   if (sides.length > 0) {
-    args.push('--projects', ...sides)
+    jestArgs.push('--projects', ...sides)
   }
 
-  try {
-    const cacheDirDb = `file:${ensurePosixPath(CACHE_DIR)}/test.db`
-    const DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
+  const cacheDirDb = `file:${ensurePosixPath(CACHE_DIR)}/test.db`
+  const DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
```

Comment:
I've made a suggestion to move this closer to the action.
```suggestion
```

Reply:
We can't do this because DATABASE_URL is used again in yarn jest. Not sure why, but hey, if it aint broke...